src/System.Drawing.Common/src/System/Drawing/Font.cs

```diff
@@ -368,12 +368,7 @@ public static Font FromLogFont(object lf)
         public static Font FromLogFont(object lf, IntPtr hdc)
         {
             IntPtr font = IntPtr.Zero;
-            int status;
-
-            if (Marshal.SystemDefaultCharSize == 1)
-                status = SafeNativeMethods.Gdip.GdipCreateFontFromLogfontA(new HandleRef(null, hdc), lf, out font);
```

Comment:
I think you can remove some other declarations for "a" versions also eg GetTextMetricsA is declared in two places. I see 8 hits on "extern.*A\("